src/tests/unit/test-data.js

```diff
@@ -561,17 +561,36 @@ const serializePizzaData = {
+  FROM PIZZAS 
+  LEFT JOIN DOUGHS ON PIZZAS.DOUGH_ID = DOUGHS.ID
+  LEFT JOIN PIZZA_INGREDIENTS ON PIZZAS.ID = PIZZA_INGREDIENTS.PIZZA_ID
+  LEFT JOIN INGREDIENTS ON INGREDIENTS.ID = PIZZA_INGREDIENTS.INGREDIENT_ID
+  WHERE PIZZAS.ID = :id`,
+  getPizzaIngredientsQuery: `SELECT PIZZAS.ID AS "PIZZA_id", PIZZAS.DOUGH_ID AS "PIZZA_doughId", PIZZAS.NAME AS "PIZZA_name", PIZZAS.BAKE_TIME AS "PIZZA_bakeTime", PIZZAS.OVEN_TEMP AS "PIZZA_ovenTemp", PIZZAS.SPECIAL_INSTRUCTIONS AS "PIZZA_specialInstructions", INGREDIENTS.ID AS "INGREDIENT_id", INGREDIENTS.TYPE AS "INGREDIENT_ingredientType", INGREDIENTS.NAME AS "INGREDIENT_name", INGREDIENTS.NOTES AS "INGREDIENT_notes"
+  FROM PIZZAS 
+  
+  LEFT JOIN PIZZA_INGREDIENTS ON PIZZAS.ID = PIZZA_INGREDIENTS.PIZZA_ID
+  LEFT JOIN INGREDIENTS ON INGREDIENTS.ID = PIZZA_INGREDIENTS.INGREDIENT_ID
+  WHERE PIZZAS.ID = :id`,
+  getPizzaDoughQuery: `SELECT PIZZAS.ID AS "PIZZA_id", PIZZAS.DOUGH_ID AS "PIZZA_doughId", PIZZAS.NAME AS "PIZZA_name", PIZZAS.BAKE_TIME AS "PIZZA_bakeTime", PIZZAS.OVEN_TEMP AS "PIZZA_ovenTemp", PIZZAS.SPECIAL_INSTRUCTIONS AS "PIZZA_specialInstructions", DOUGHS.ID AS "DOUGH_id", DOUGHS.NAME AS "DOUGH_name", DOUGHS.GRAMS_FLOUR AS "DOUGH_gramsFlour", DOUGHS.GRAMS_WATER AS "DOUGH_gramsWater", DOUGHS.FLOUR_TYPE AS "DOUGH_flourType", DOUGHS.WATER_TEMP AS "DOUGH_waterTemp", DOUGHS.GRAMS_YEAST AS "DOUGH_gramsYeast", DOUGHS.GRAMS_SALT AS "DOUGH_gramsSalt", DOUGHS.GRAMS_SUGAR AS "DOUGH_gramsSugar", DOUGHS.GRAMS_OLIVE_OIL AS "DOUGH_gramsOliveOil", DOUGHS.BULK_FERMENT_TIME AS "DOUGH_bulkFermentTime", DOUGHS.PROOF_TIME AS "DOUGH_proofTime", DOUGHS.SPECIAL_INSTRUCTIONS AS "DOUGH_specialInstructions"
+  FROM PIZZAS 
+  LEFT JOIN DOUGHS ON PIZZAS.DOUGH_ID = DOUGHS.ID
+  
+  WHERE PIZZAS.ID = :id`,
+
+  getPizzaQuery: `SELECT PIZZAS.ID AS "PIZZA_id", PIZZAS.DOUGH_ID AS "PIZZA_doughId", PIZZAS.NAME AS "PIZZA_name", PIZZAS.BAKE_TIME AS "PIZZA_bakeTime", PIZZAS.OVEN_TEMP AS "PIZZA_ovenTemp", PIZZAS.SPECIAL_INSTRUCTIONS AS "PIZZA_specialInstructions"
+  FROM PIZZAS 
```

Comment:
Is this necessary for an automatically generated query? I understand the value of doing this for queries that are written in code by hand, but in this case we'd be changing our query generation logic just to make the unit test sample data look better.

Reply:
I guess yes. My reason is that even test data is also the code that we need to maintain in the future. It's not only to make the code look prettier, but also help others who maintain your code easier. That's also probably why I'm a little hesitated to write functions to generate most part of SQL queries automatically; it's kind of making the query implicitly to me and could potentially increase the difficulty of debugging in the future. 

When I first review your code, I felt like using functions to automatically generate query string was really smart, but I wasn't sure will there be any side effect. The reason I didn't bring this up was since nobody did this before and I was really curious how things work :)

Reply:
That makes sense. One issue I've run into, on the subject of maintainability, is that when you're writing tests that check what queries were run against the database you have to do string comparisons on the queries, so differences in whitespace can cause the comparison to fail even if the query itself is actually the same. From that perspective, more readable SQL can be less maintainable since it can introduce errors that are harder to troubleshoot in the tests.

That said, I'm working on updating the style used in the SQL query right now to split up the aliases across multiple lines, I just wanted to mention this since that issue came up while I was working.

Reply:
True, I also ran into this issue in my personal side project before. I recalled that I eventually used [dedent](https://docs.python.org/3.1/library/textwrap.html#textwrap.dedent) function (in Python, but I belive there should be something similar in JS) to solve the issue. So maybe you can look into this and see if that help?

Reply:
Thanks, that's a really good suggestion. I'll look into that since that would solve the only real issue I had. My dream is a SQL parser that would let us actually compare the semantics of the two SQL statements but I don't think that exists, sadly.

Reply:
well, I don't know will this module help, but looks like it's really popular and many people use it to build SQL query: https://knexjs.org/

Reply:
Interesting, that might be something to look into. I was thinking something more like [this](https://www.npmjs.com/package/js-sql-parser) that can generate an AST from an existing query. But using a full on query generator might make sense too.

Reply:
@tsoliangwu0130  got this updated after experimenting with a couple different strategies ([this](https://www.npmjs.com/package/flora) gets very close to being able to do AST diffs but throws an error on some valid oracle syntax), finally settled on just stripping extra whitespace out of the queries after having trouble getting `dedent` to work. Let me know what you think.

Reply:
This is officially fixed and dedent is working.